README.md

```diff
@@ -2,39 +2,51 @@
 A curated list of utilities we use on mac. This guide will help you to set up your development environment.
 
 ### First things first
-1. Chrome
-2. Brave
+1. [Chrome](https://www.google.com/chrome/?brand=CHBD&gclsrc=ds&gclsrc=ds)
+2. [Brave](https://brave.com/download/)
 
 ### Communication
-1. Slack
-2. Discord
+1. [Slack](https://slack.com/intl/en-in/downloads/mac)
+   -AppStore [Slack](https://apps.apple.com/app/slack/id803453959?ls=1&mt=12)
```

Comment:
we can remove this. there's already a link in their website for app store

Reply:
Okay